dev/src/hal/LedMatrix.cpp

```diff
@@ -16,34 +16,26 @@
 #include "mcal/Gpio.h"
 #include "mcal/Systick.h"
 #include "hal/LedMatrix.h"
-#include "utils/Types.h"
 
+using namespace stm32::util;   
+using namespace stm32::type;
+using namespace stm32::font;
 using namespace stm32::dev::mcal::pin;
 using namespace stm32::dev::mcal::gpio; 
-using namespace stm32::util;
-using namespace stm32::type;
 using namespace stm32::dev::mcal::systick;       
-using namespace stm32::utils::font;
 using namespace stm32::dev::hal::ledMatrix;
 
-#define ACTIVE_VOLT_STATE(connectionType)    ((connectionType) == kCommon_Row_Anode ? kHigh : kLow)     //  NOLINT  
-#define DEACTIVE_VOLT_STATE(connectionType)  ((connectionType) == kCommon_Row_Anode ? kLow  : kHigh)    //  NOLINT
+#define ACTIVE_VOLT_STATE(connectionType)    (connectionType == kCommon_Row_Anode ? kHigh : kLow)  
+#define DEACTIVE_VOLT_STATE(connectionType)  (connectionType == kCommon_Row_Anode ? kLow  : kHigh) 
 
-template <MatrixConnectionType connectionType>
+template <LedMatrixConnection connectionType>
 LedMatrix<connectionType>::LedMatrix(const Array_t rowPins, const Array_t colPins) 
     : rowPins_(rowPins) , colPins_(colPins) {} 
 
-template <MatrixConnectionType connectionType>
+template <LedMatrixConnection connectionType>
 void LedMatrix<connectionType>::Init() {
     // SET [ROW & COL] PINS AC OUTPUT [PUSHPULL]
```

Comment:
@gehadHassanin In line 39 "PinMode GetPinMode();" , could you explain its purpose? I'm wondering if it's a mistake.